### DISCUSSION TITLE
Error: java.lang.NullPointerException: Cannot invoke "String.equals(Object)" because "accountsetupbutton" is null

### DISCUSSION THREAD

**Discussion by vitorpadilha** [exoplatform/exo-community#40] [Q&A]
Hello,

In this Week this error was appear when i try create a user using Docker with image "exoplatform/exo:6.5_latest" and "exoplatform/exo:6.4_latest". 

The service start but when i try create a new user with root this message shown: "java.lang.NullPointerException: Cannot invoke "String.equals(Object)" because "accountsetupbutton" is null" and I can't start to use the platform.

Is it happen with someone more? Someone can help me?

Thanks.

**Comment by rdenarie** [comment]
Hello @vitorpadilha 

I reproduce the problem with 6.4_latest and 6.5_latest docker image.
I will create an issue for the maintenance team.

In order to make it work, you can skip the Account Form. 
Then you can login with the default user root/password
With this user, you will be able to go in administration->User Management and create the user you want.

**Comment by rdenarie** [comment]
PR resolving this issue : exoplatform/platform-private-distributions#327
